tests/lib/internal-rules/internal-no-valid-meta.js

```diff
@@ -0,0 +1,263 @@
+/**
+ * @fileoverview Tests for internal no-valid-meta rule.
+ * @author Vitor Balocco
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/internal-rules/internal-no-invalid-meta"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+var ruleTester = new RuleTester();
+
+ruleTester.run("internal-no-invalid-meta", rule, {
+    valid: [
+
+        // context.report() call with no fix
+        [
+            "module.exports = {",
+            "    meta: {",
+            "        docs: {",
+            "            description: 'some rule',",
+            "            category: 'Internal',",
+            "            recommended: false",
+            "        },",
+            "        schema: []",
+            "    },",
+
+            "    create: function(context) {",
+            "        return {",
+            "            Program: function(node) {",
+            "                context.report({",
+            "                    node: node",
+            "                });",
+            "            }",
+            "        };",
+            "    }",
+            "};"
+        ].join("\n"),
```

Comment:
I think template literals using an approach similar to that `stripIndent` template tag makes a lot of sense.
Ideally we could refactor all of the tests to take advantage of it, so I think perhaps it's out of the scope of this issue. What do you guys think?